app/actors/hyrax/default_admin_set_actor.rb

```diff
@@ -35,7 +35,7 @@ def default_exists?
       # Creates the default AdminSet and an associated PermissionTemplate with workflow
       def create_default_admin_set
         AdminSet.create!(id: DEFAULT_ID, title: ['Default Admin Set']).tap do |_as|
-          PermissionTemplate.create!(admin_set_id: DEFAULT_ID, workflow_name: 'default')
+          PermissionTemplate.create!(admin_set_id: DEFAULT_ID, workflow_id: 1)
```

Comment:
Not liking this; that is a more magical number than the magic string 'default'

How about:

```ruby
  PermissionTemplate.create!(admin_set_id: DEFAULT_ID, workflow_id: default_workflow.id)

# @raises ActiveRecord::RecordNotFound when we don't have a workflow
def default_workflow
  Sipity::Workflow.order('id ASC').first!
end
```

This will throw an exception if it does